R/expr-interp.R

```diff
@@ -13,30 +13,42 @@
 #' expression. We provide both syntactic operators and functional
 #' forms for unquoting.
 #'
-#' - `UQ()` and the `!!` operator unquote their argument. It gets
-#'   evaluated immediately in the surrounding context.
+#' - The `!!` operator unquotes its argument. It gets evaluated
+#'   immediately in the surrounding context.
 #'
-#' - `UQE()` is like `UQ()` but retrieves the expression of
-#'   [quosureish][is_quosureish] objects. It is a shortcut for `!!
-#'   get_expr(x)`. Use this with care: it is potentially unsafe to
-#'   discard the environment of the quosure.
+#' - The `!!!` operator unquotes and splices its argument. The
+#'   argument should represents a list or a vector. Each element will
+#'   be embedded in the surrounding call, i.e. each element is
+#'   inserted as an argument. If the vector is named, the names are
+#'   used as argument names.
 #'
-#' - `UQS()` and the `!!!` operators unquote and splice their
-#'   argument. The argument should evaluate to a vector or an
-#'   expression. Each component of the vector is embedded as its own
-#'   argument in the surrounding call. If the vector is named, the
-#'   names are used as argument names.
 #'
 #' @section Theory:
 #'
-#' Formally, `quo()` and `expr()` are quasiquote functions, `UQ()` is
-#' the unquote operator, and `UQS()` is the unquote splice operator.
+#' Formally, `quo()` and `expr()` are quasiquote functions, `!!` is
+#' the unquote operator, and `!!!` is the unquote splice operator.
 #' These terms have a rich history in Lisp languages, and live on in
 #' modern languages like
 #' [Julia](https://docs.julialang.org/en/stable/manual/metaprogramming/)
 #' and
 #' [Racket](https://docs.racket-lang.org/reference/quasiquote.html).
 #'
+#'
+#' @section Life cycle:
+#'
+#' * `UQ()` and `UQS()` were soft-deprecated in rlang 0.2.0 in order
```

Comment:
I think you also need to note here that the prefix forms aren't as important now because of the other changes to the syntax (i.e. operator precedence and parens stripping).